Makefile

```diff
@@ -13,6 +13,8 @@ ORG := jenkins-x
 ORG_REPO := $(ORG)/$(NAME)
 RELEASE_ORG_REPO := $(ORG_REPO)
 ROOT_PACKAGE := github.com/$(ORG_REPO)
+
+CATALOG_GO_VERSION := 1.17.8
```

Comment:
It's not 1.17.8. This is where the value is: https://github.com/jenkins-x/jx3-pipeline-catalog/blob/f46083677cfc51f6e6e2f04af5ed2ab6b5068719/tasks/go/pullrequest.yaml#L37

Reply:
@ankitm123 Yes, I was trying to see if checks will fail so I made it something different. But my question is if this value should be from this file https://github.com/jenkins-x/jx3-pipeline-catalog/blob/f46083677cfc51f6e6e2f04af5ed2ab6b5068719/tasks/go/pullrequest.yaml
or it can be hardcoded in the make file

Reply:
It's from that file. We can use updatebot to change the catalog_version in makefile.